pages/index.js

```diff
@@ -1,14 +1,16 @@
 import Head from 'next/head'
+import InfoCard from '../components/InfoCard'
 import Navbar from '../components/Navbar'
 import Footer from '../components/Footer'
 
-export default function Home () {
+export default function Home() {
   return (
     <div>
       <Head>
         <title>Kodemia</title>
         <link rel='icon' href='/favicon.ico' />
       </Head>
+      <InfoCard icon='/icons/Icon-Objetivo.svg' title='Te acompañamos en el proceso' description='Todos los días, por módulos y por Bootcamp, llevarás proyectos prácticos reales' />
```

Comment:
Esta linea esta muy larga, hay que partirla

```suggestion
      <InfoCard 
        icon='/icons/Icon-Objetivo.svg' 
        title='Te acompañamos en el proceso'
        description='Todos los días, por módulos y por Bootcamp, llevarás proyectos prácticos reales'
      />
```